app/javascript/components/sessions/SignIn.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+async function createSession(sessionUser) {
+  const sessionUrl = '/api/v1/sessions';
+
+  await fetch(sessionUrl, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      status: 'success',
+      user: {
+        email: sessionUser.email,
+        password: sessionUser.password,
+      },
+    }),
+  })
+    .then((response) => {
+      if (response.status === 204) {
+        // TODO: samuel - Need to discuss how to Authenticate the user
+        // Do Something
+      } else {
+        // Do Something
+      }
+    })
+    .catch((error) => {
+      console.log('json response is error', error);
+    });
+}
+
+export default function SignIn() {
+  type FormInputs = {
+    email: string;
+    password: string;
+    auth: boolean;
+  };
+
+  const { register, handleSubmit, formState: { errors } } = useForm<FormInputs>();
+
+  const handleSignIn = (sessionUser) => createSession(sessionUser);
+
+  return (
+    <div className="col-md-3 mx-auto">
+      <h1> Sign In </h1>
+      <Form onSubmit={handleSubmit(handleSignIn)}>
```

Comment:
I think it's better to have a single source of truth so we can disable the HTML5 validations here by using the `noValidate` prop and set it to `true`.
If you like to disable the browser auto completion we can also use `autoComplete="off"`.

Reply:
I don't understand why we should remove front end validations and auto completion

Reply:
Oh, didn't mean that we should remove front end validations.
HTML 5 by default has some basic input validations with the explicit use of some build in attributes (i.e. required) and also implicitly depending of the type of the input (i.e. an input with type="email" will automatically get validated to match the HTML 5 pattern for emails).
This has nothing to do with JavaScript, it may feel good but this can introduce some troubles because will have two sources of validations one that we made and know (the validations through Yup, or react-hook-form or custom made by our JavaScript code) and those of HTML5.